pandas/tests/indexes/period/test_period_range.py

```diff
@@ -51,6 +51,16 @@ def test_construction_from_period(self):
         result = period_range(start=start, end=end, freq="Q", name="foo")
         tm.assert_index_equal(result, expected)
 
+        # test for issue #21793
+        idx = period_range(start=start, end=end, freq="Q", name="foo")
+        result = idx.values
+        expected = np.array([Period('2017Q1', 'Q-DEC'),
```

Comment:
We want to be testing the result of the `idx == idx.values` comparison specifically.

Reply:
Should be changed now, I appreciate your patience :)